debezium-operator-api/src/main/java/io/debezium/operator/api/model/FormatType.java

```diff
@@ -19,6 +20,7 @@ public class FormatType implements ConfigMappable<DebeziumServer> {
 
     @JsonPropertyDescription("Format type recognised by Debezium Server.")
     @JsonProperty(defaultValue = "json")
+    @Default("json")
```

Comment:
@mfvitale the default wasn't generated until now, it doesn't have to be. We will be removing this once again when there is a proper fix  in fabric8.

We can either ignore it and wait. Or add the @Default into docs generator and replace the JsonProperty (though since for other things @JsonPropertyDescription is still used, I would stick with Jackson annotations for now).